action.yaml

```diff
@@ -64,8 +65,9 @@ runs:
             )
         for host_address in itertools.islice(network, 1, None):
             if host_address != address:
-                print("::set-output name=devstack-ip::{}".format(address))
-                print("::set-output name=host-address::{}/{}".format(host_address, prefix_len))
+                with open(os.environ['GITHUB_OUTPUT'], "a") as f:
```

Comment:
should we rename `f` to `file` to make it easier to read?

Reply:
Done :)